server/src/test/resources/logback-test.xml

```diff
@@ -1 +1,24 @@
-<root level="error"> <appender-ref ref="console"/> </root>
+<configuration>
+    <variable name="ENV_LOG_LEVEL" value="${ENV_LOG_LEVEL:-INFO}" />
```

Comment:
we either need to specify the log level in drone, or make this default to ERROR.